### ISSUE TITLE
Got WARNING and ERROR info when using StrelkaGermlineWorkflow on multi-samples

### ISSUE THREAD

**Issue by fredsamhaak** [Illumina/strelka#67]
Hi, 

I simply run Strelka2 using Strelka**Germline**Workflow on **multi-samples** and got two types of **WARNING** info: 

1. [2018-09-10T03:34:11.842427Z] [localhost.localdomain] [21220_1] [StrelkaGermlineWorkflow] [WARNING] Adaptive indel error rate estimation for sample 'default' did not succeed, using static indel error model instead.
2. [2018-09-10T03:34:13.226335Z] [localhost.localdomain] [21220_1] [TaskRunner:CallGenome+callGenomeSegment_chromId_001_chr1_0001] [WARNING] Task: 'CallGenome+callGenomeSegment_chromId_001_chr1_0001' failed but qualifies for retry. Total task failures (including this one): ...

and there are also **ERROR** info: 

[2018-09-10T03:37:14.146220Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] Failed to complete command task: 'CallGenome+callGenomeSegment_chromId_001_chr1_0000' launched from sub-workflow 'CallGenome', error code: ...

This may be a simple and stupid question but I have no idea how to deal with it.

My command lines are as follows: 

/home/shenhe/software/strelka-2.9.2.centos6_x86_64/bin/configureStrelkaGermlineWorkflow.py \
--bam out.HiSeqX101700034-CTCAATGA_HCKVKCCXY_L3.sorted.bam \
--bam out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.sorted.bam \
--ref ../00.ref/ucsc.hg19.mhc.fasta \
--runDir TEST2SAMPLES_germline

TEST2SAMPLES_germline/runWorkflow.py -m local -j 20

As for the bam files, I got them by simply running "bwa-mem&samtools"(Use out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.sorted.bam as an example): 

bwa mem ../00.ref/ucsc.hg19.mhc.fasta ../01.FastqPreparedForStrelka2/out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3_1.fq.gz ../01.FastqPreparedForStrelka2/out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3_2.fq.gz | samtools view -bS --thread 5 - > out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.bam

samtools sort out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.bam -o out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.sorted.bam

samtools index out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.sorted.bam

What makes me confused is that I use the SAME Strelka germline pipeline for ONE sample but no exceptions are thrown.

May I know the solution for this problem?

Thank you very much!

**Comment by ctsa** [comment]
Hi, Sorry you're having trouble. Could you include the full log for this case? There should typically be more specific error details adjacent to the error line:
```
[2018-09-10T03:37:14.146220Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] Failed to complete command task: 'CallGenome+callGenomeSegment_chromId_001_chr1_0000' launched from sub-workflow 'CallGenome', error code: ...
```

**Comment by fredsamhaak** [comment]
I'm sorry because it may be a little difficult to read the **ERROR info** below: 

**[TaskRunner:CallGenome+callGenomeSegment_chromId_001_chr1_0003]** Task initiated on local node
**[2018-09-10T03:37:14.146220Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR]** Failed to complete command task: 'CallGenome+callGenomeSegment_chromId_001_chr1_0000' launched from sub-workflow 'CallGenome
', error code: 1, command: '/home/shenhe/software/strelka-2.9.2.centos6_x86_64/libexec/starling2 --region chr1:1-11869078 --ref /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/00.ref/ucsc
.hg19.mhc.fasta --max-indel-size 49 --min-mapping-quality 20 --gvcf-output-prefix /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/genomeSegme
nt.tmpdir/segment.chromId_001_chr1_0000. --gvcf-min-gqx 15 --gvcf-min-homref-gqx 15 --gvcf-max-snv-strand-bias 10 --enable-read-backed-phasing --stats-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCal
ling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/genomeSegment.tmpdir/runStats.chromId_001_chr1_0000.xml --snv-scoring-model-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/config/ge
rmlineSNVScoringModels.json --indel-scoring-model-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/config/germlineIndelScoringModels.json --align-file /data/shenhe/02.VariantsCalling/00.Strelka2Var
iantsCalling/00.germline/03.tensamples/out.HiSeqX101700034-CTCAATGA_HCKVKCCXY_L3.sorted.bam --align-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/out.HiSeqX101700034-
CTGAGCCA_HCKVKCCXY_L3.sorted.bam --gvcf-skip-header --chrom-depth-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/chromDepth.tsv --indel
-error-models-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/sequenceErrorModel.Sample000.json --indel-error-models-file /data/shenhe/0
2.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/sequenceErrorModel.Sample001.json --theta-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/con
fig/theta.json'
**[2018-09-10T03:37:14.182107Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** Error Message:
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** Last 8 stderr lines from task (of 8 total lines):
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.965052Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** FATAL_E
RROR: 2018-Sep-10 11:37:13 /builder/src/c++/lib/calibration/IndelErrorModel.cpp(209): Throw in function IndelErrorModel::IndelErrorModel(const std::vector<std::basic_string<char> >&, const string&, const std::v
ector<std::basic_string<char> >&)
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.972188Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** Dynamic
 exception type: boost::exception_detail::clone_impl<illumina::common::GeneralException>
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.979047Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** std::ex
ception::what: Failed to find indel error model for '/data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.sorted.bam'
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.985502Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** 
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.992082Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** cmdline
:     /home/shenhe/software/strelka-2.9.2.centos6_x86_64/libexec/starling2 --region chr1:1-11869078 --ref /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/00.ref/ucsc.hg19.mhc.fasta --max-
indel-size 49 --min-mapping-quality 20 --gvcf-output-prefix /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/genomeSegment.tmpdir/segment.chro
mId_001_chr1_0000. --gvcf-min-gqx 15 --gvcf-min-homref-gqx 15 --gvcf-max-snv-strand-bias 10 --enable-read-backed-phasing --stats-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.te
nsamples/TEST2SAMPLES_germline/workspace/genomeSegment.tmpdir/runStats.chromId_001_chr1_0000.xml --snv-scoring-model-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/config/germlineSNVScoringModels
.json --indel-scoring-model-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/config/germlineIndelScoringModels.json --align-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germli
ne/03.tensamples/out.HiSeqX101700034-CTCAATGA_HCKVKCCXY_L3.sorted.bam --align-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/out.HiSeqX101700034-CTGAGCCA_HCKVKCCXY_L3.
sorted.bam --gvcf-skip-header --chrom-depth-file /data/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/chromDepth.tsv --indel-error-models-file /da
ta/shenhe/02.VariantsCalling/00.Strelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/sequenceErrorModel.Sample000.json --indel-error-models-file /data/shenhe/02.VariantsCalling/00.S
trelka2VariantsCalling/00.germline/03.tensamples/TEST2SAMPLES_germline/workspace/sequenceErrorModel.Sample001.json --theta-file /home/shenhe/software/strelka-2.9.2.centos6_x86_64/share/config/theta.json
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:13.998099Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** version
:     2.9.2
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:14.004287Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** buildTi
me:   2018-03-02T22:08:15.960987Z
**[2018-09-10T03:37:14.218116Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR] [2018-09-10T03:37:14.012179Z] [localhost.localdomain] [21220_1] [CallGenome+callGenomeSegment_chromId_001_chr1_0000]** compile
r:    g++-6.3.1
**[2018-09-10T03:37:14.244305Z] [localhost.localdomain] [21220_1] [TaskManager] [ERROR]** Shutting down task submission. Waiting for remaining tasks to complete.
**[2018-09-10T03:37:14.267954Z] [localhost.localdomain] [21220_1] [TaskRunner:CallGenome+callGenomeSegment_chromId_000_chrM_0000]** Retrying task: 'CallGenome+callGenomeSegment_chromId_000_chrM_0000'. Total prior t
ask failures: 2

**Comment by sangtaekim** [comment]
Thanks @fredsamhaak! This issue has been fixed in the v2.9.5 release (https://github.com/Illumina/strelka/releases/tag/untagged-839da48539154c23a780). Please use v2.9.5 or greater.

**Comment by fredsamhaak** [comment]
Thanks @sangtaekim and @ctsa ! 

I analyse 10 samples using multi-samples StrelkaGermlineWorkflow of Strelka v2.9.5 and it works!
I only get WARNING info showing that adaptive indel error rate cannot be estimated in some of these samples but it seems that it is no big deal. Thanks again!
